client/src/pages/EditAccount.js

```diff
@@ -138,7 +138,7 @@ function EditAccount(props) {
         );
       }
     })();
-  }, [userProfileDispatch]);
+  }, [t, userProfileDispatch]);
```

Comment:
could we give a more descriptive name here for `t`? Could we change it to something else? Something more descriptive.

Reply:
I didn't made those changes when i merge staging, then eslint added this function as we are using it inside useEffect. `t` function is for the translation function provided by i18n.